lib/ProductOpener/Display.pm

```diff
@@ -264,6 +273,21 @@ sub init()
 		return 301;
 	}
 	
+	
+	# Allow cc and lc overrides as query parameters
```

Comment:
The intended use is for the API requests, we'll probably document it only for the API (even if it does work on other requests too)